Entrega Proyecto 1

joaquinrau opened the pull request:
### Entrega Proyecto 1

- Tema Elegido: 2 
- Link a GitHub Pages: [link](https://joaquin699.github.io/proyecto-1/) 

- Link al video de la explicacion: [video](https://drive.google.com/open?id=1XMmqod4QOXN8WXp61ODJw0z46KM6Okvk)

joaquinrau committed:
Estructura HTML y CSS basica para nivel 1.

joaquinrau committed:
Creacion de tablero automatica en base al nivel seleccionado.
Funcionamiento del Cronometro para el puntaje.

joaquinrau committed:
Carpeta con imagenes a utilizar en el juego

joaquinrau committed:
Logica del juego terminada
Funcionamiento de todos los botones (niveles, start, stop)

joaquinrau committed:
Logica para finalizar el juego y guardar la puntuacion
Diseño del juego terminado

joaquinrau committed:
Agrego fuentes

joaquinrau committed:
Separe archivos css

joaquinrau committed:
Funcionalidad para cambiar de temas

joaquinrau committed:
Segundo color del juego terminado

joaquinrau committed:
Correccion errores

joaquinrau committed:
Refactor javascript

joaquinrau committed:
Añadi Icon y titulo

joaquinrau committed:
Refactor y correccion de errores

MarianoTucat reviewed (approved):
# Proyecto 1 - Rau

1. HTML: Muy Bien
2. CSS: Bien
3. Javascript: Muy Bien
4. Manejo DOM: Muy Bien
5. Visualmente Atractiva: Bien
6. Local Storage: Regular
7. PR a tiempo: Si
8. GitHub Pages: Si
9. Video: Bien
Comentarios: Buen proyecto. Detalles de CSS, el cursor pierde el ícono pointer en los botones cuando se pasa por sobre el texto. Detalles de Local Storage, era más simple grabar un objeto JSON convertido a string, y no todos los valores planos y tener que reacomodar manualmente los valores al cambiar la tabla de posiciones. 
Estrellas: 🌟🌟🌟🌟
Resultado: Aprobado

joaquinrau committed:
Modifique forma de almacenar los datos en local Storage.
El cursor no pierde el icono pointer cuando se pasa sobre las letras de un boton

joaquinrau committed:
Fix bug al presionar boton restart
